### DISCUSSION TITLE
DNS Name in IP Addresses

### DISCUSSION THREAD

**Discussion by JHarrison712** [netbox-community/netbox#12868] [General]
Good afternoon,

I been assigning my public ip address inside netbox and I found that you can only put in 1 DNS entire per IP. However I believe some people reuses the same public ip address just different A records. Does anyone know how to get around this?

**Comment by candlerb** [comment]
Yes: you need to manage your additional DNS records externally from Netbox, or use a third-party DNS management plugin in Netbox. The same applies if you want to manage other record types like MX, CNAME, TXT etc.

The dns_name field in Netbox is just for the primary name for that IP address, i.e. the one which includes a PTR record for the reverse DNS.

Whilst it is common to have many names pointing to the same IP address, you shouldn't have multiple PTR records for an IP address (which would effectively pick one at random using round-robin DNS)

**Reply by JHarrison712** [reply]
Good to know thank you.
